README.md

```diff
@@ -1 +1,54 @@
-# lookout
\ No newline at end of file
+# lookout [![Build Status](https://travis-ci.org/src-d/lookout.svg)](https://travis-ci.org/src-d/lookout) ![GoDoc](https://godoc.org/gopkg.in/src-d/lookout?status.svg)](https://godoc.org/github.com/src-d/lookout)
+
+A service for assisted code review, allows running custom code analysis on PRs.
+
+# Installation
+
+```
+Short installation guide or link to longer installation instructions.
+This should include a pre-requisites subsection if needed.
+
+Are there Docker images, packages managers (brew, apt, etc), installations scripts?
+```
+
+# Example
+
+## SDK
+
+If you are developing an Analyzer, please check [SDK documentation](./sdk/README.md).
+It includes curl-style binary that allows to trigger Analysers directly, without launchin a full lookout server.
+
+## Server
+
+To run a lookout server with defatult, dummy analyzer.
+
+### Local
+
+To trigger the analysis on actual PR of your Github repository do:
+
+1. Start an analyzer
+Any of the analysers, or a default dummy one, included in this repository
+  ```
+  go build -o analyzer ./cmd/dummy
+  ./dummy
+  ```
+1. Start a lookout server
+  `lookout serve <repository>`
```

Comment:
it would fail without tokens. We should add information about flags/env vars or `--dry-run` flag (works without token)